version/features.go

```diff
@@ -17,15 +17,20 @@ package version
 
 // Versions list.
 var (
-	V3_0_0 = MustParse("3.0.0-0") //nolint:revive,stylecheck
+	V2_36_0 = MustParse("2.36.0-0") //nolint:revive,stylecheck
+	V3_0_0  = MustParse("3.0.0-0")  //nolint:revive,stylecheck
+	// TODO: change to 3.2.0 when 3.1.0 is released.
+	V3_2_0 = MustParse("3.1.0-0") //nolint:revive,stylecheck
 )
 
 // FeatureVersion represents a minimum version feature being supported.
 type FeatureVersion *Parsed
 
 // Features list.
 var (
-	NodeExporterNewTLSConfigVersion FeatureVersion = V3_0_0
+	NodeExporterNewTLSConfig     FeatureVersion = V3_0_0
+	MysqlExporterMySQL8_4        FeatureVersion = V3_2_0
```

Comment:
maybe it's better to base naming on mysqld_exporter version instead of MySQL version as we are going to use the same config for all MySQL versions.

Reply:
Could be...
Done.